Documentation/ServiceUtilities/MixedRealityServiceRegistryAndIMixedRealityServiceRegistrar.md

```diff
@@ -0,0 +1,54 @@
+# What are the MixedRealityServiceRegistry and IMixedRealityServiceRegistrar
+
+The Mixed Reality Toolkit has two very similarly named components that perform related tasks: 
+MixedRealityServiceRegistry and IMixedRealityServiceRegistrar. What are they and what purpose does each serve?
```

Comment:
Remove "What are they and what purpose do they serve" redundant with the title

Reply:
removed